acceptance-tests/src/test/java/io/blueocean/ath/offline/BitbucketServerTest.java

```diff
@@ -0,0 +1,66 @@
+package io.blueocean.ath.offline;
+
+import com.cdancy.bitbucket.rest.BitbucketClient;
+import com.cdancy.bitbucket.rest.options.CreateRepository;
+import com.google.inject.Inject;
+import io.blueocean.ath.ATHJUnitRunner;
+import io.blueocean.ath.Login;
+import io.blueocean.ath.WaitUtil;
+import io.blueocean.ath.pages.blue.DashboardPage;
+import org.apache.log4j.Logger;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.io.IOException;
+
+@Login
+@RunWith(ATHJUnitRunner.class)
+public class BitbucketServerTest {
+
+    private static Logger LOGGER = Logger.getLogger(BitbucketServerTest.class);
+
+    @Inject
+    WaitUtil wait;
+
+    @Inject
+    DashboardPage dashboardPage;
+
+    @Test
+    public void testJenkinsfileCreate() {
+        BitbucketClient client = BitbucketClient.builder()
+            .endPoint("http://127.0.0.1:7990/bitbucket")
```

Comment:
@imeredith this wont' work on CI IIRC - as the code running the ATH runs in one docker container, the bitbucket one in a peer container. On your desktop, the BB container is running locally. So I think the Jenkinsfile needs to change to link in the network/port from BB. 

Locally this test worked for me, but not on CI.